tests/fast-integration/eventing-test/assets/eventing-function.yaml

```diff
@@ -0,0 +1,132 @@
+apiVersion: serverless.kyma-project.io/v1alpha2
+kind: Function
+metadata:
+  labels:
+    serverless.kyma-project.io/build-resources-preset: local-dev
+    serverless.kyma-project.io/function-resources-preset: S
+    serverless.kyma-project.io/replicas-preset: S
+  name: eventing-sink
+spec:
+  scaleConfig:
+    minReplicas: 1
+    maxReplicas: 1
+  runtime: nodejs16
+  resourceConfiguration:
+    function:
+      resources:
+        limits:
+          cpu: 1
+          memory: 1Gi
+        requests:
+          cpu: 500m
+          memory: 500Mi
+    build:
+      resources:
+        limits:
+          cpu: 1
+          memory: 1Gi
+        requests:
+          cpu: 500m
+          memory: 500Mi
+  source:
+    inline:
+      dependencies: |
+        {
+          "name": "orders",
+          "version": "1.0.0",
+          "dependencies": {
+            "axios": "^0.19.2"
+          }
+        }
+      source: |
+        let receviedEvents = {};
```

Comment:
can we get some inline documentation on how to use this function? This was one of the main problems with the original version, we should address it here

Reply:
good point, but I will add it in a separate PR.